src/test/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/PrisonerIepLevelReviewServiceTest.kt

```diff
@@ -265,6 +267,38 @@ class PrisonerIepLevelReviewServiceTest {
     }
   }
 
+  @Nested
+  inner class addIepMigration {
+
+    @Test
+    fun `process migration`(): Unit = runBlocking {
+      // Given
+      val bookingId = 1234567L
+      whenever(prisonApiService.getPrisonerInfo(bookingId)).thenReturn(prisonerAtLocation())
+      whenever(prisonerIepLevelRepository.save(any())).thenAnswer { i -> i.arguments[0] }
+
+      // When
+      prisonerIepLevelReviewService.addIepMigration(bookingId, iepMigration)
+
+      // Then
+      verify(prisonerIepLevelRepository, times(1)).save(
```

Comment:
This `verify(prisonerIepLevelRepository, times(1)).save(PrisonerIepLevel(...))` syntax...just to be sure I understand, this is checking that `save()` was called once (`times(1)`) on the `prisonerIepLevelRepository` is that right?

in general it just means `verify(instance, times(n_times)).method_name(args)`, check that `method_name()` was called `n_times` on `instance` with `args` arguments, is that or am I reading this wrong?